pkg/auth/dependency/webapp/template.go

```diff
@@ -16,8 +20,11 @@ const (
 	TemplateItemTypeAuthUILogoutHTML         config.TemplateItemType = "auth_ui_logout.html"
 )
 
-const defineHead = `
-{{ define "HEAD" }}
+var TemplateAuthUIHTMLHeadHTML = template.Spec{
+	Type:   TemplateItemTypeAuthUIHTMLHeadHTML,
+	IsHTML: true,
+	Default: `
+{{ define "auth_ui_html_head.html" }}
```

Comment:
I think it's better to have a fallback for client_name, in case user entered the account page directly? We can do this in another issue.

Reply:
Actually the placement of `client_name` is tentative. It does not appear in the design. Let me open another issue to track this.